features/release_calendar_response.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Proxy returns response from Release Calendar
+
+    The proxy can be called with Release Calendar URLs (for instance, "/releases/adoption"),
+    when this happens we want to ensure that the proxy forwards the URL to the Release Calendar. 
+
+  Scenario Outline: Proxy returns response from Release Calendar
```

Comment:
There are no examples in this scenario, so this should be simply `Scenario`, not `Scenario Outline`.